--- XaoS/configure.in.orig	2006-07-13 08:01:46.000000000 +0200
+++ XaoS/configure.in	2006-10-19 19:31:54.018598250 +0200
@@ -843,7 +843,7 @@
 if test "x$ac_cv_lib_vga_vga_hasmode$ac_cv_lib_vgagl_gl_write" = xyesyes; then
   drivers=yes
   AC_DEFINE(SVGA_DRIVER)
-  STICKY='chmod +s $(DESTDIR)$(bindir)/xaos; chown root $(DESTDIR)$(bindir)/xaos; chgrp root $(DESTDIR)$(bindir)/xaos'
+  STICKY=''
   DRIVERDIRS="$DRIVERDIRS ui/ui-drv/svga"
   DRIVERLIBS="$DRIVERLIBS lib/libuisvga.a"
   if test "x$ac_cv_func_pthread_sighandler" = xyes; then 
