--- XaoS/doc/xaos.texinfo.orig	2006-07-09 10:56:57.000000000 +0200
+++ XaoS/doc/xaos.texinfo	2006-10-19 20:16:35.698192750 +0200
@@ -11,9 +11,9 @@
 
 @setfilename xaos.info
 @settitle An fast realtime interactive fractal zoomer --- user's manual
-@dircategory Graphics
+@dircategory Graphics Utilities:
 @direntry 
- * XaoS: (xaos).               A fast real-time interactive fractal zoomer
+ * XaoS: (xaos).			A fast real-time interactive fractal zoomer
 @end direntry
 
 
--- XaoS/doc/xaosdev.texinfo.orig	2005-02-05 13:23:28.000000000 +0100
+++ XaoS/doc/xaosdev.texinfo	2006-10-19 20:17:13.488554500 +0200
@@ -7,10 +7,9 @@
 
 @setfilename xaosdev.info
 @settitle A fast real-time interactive fractal zoomer --- hacker's guide
-@dircategory Graphics
+@dircategory Graphics Utilities:
 @direntry 
- * XaoS: (xaosdev).               The fast real-time interactive fractal zoomer
-                                  (developers documentation)
+ * XaoS: (xaosdev).			The fast real-time interactive fractal zoomer (developers documentation)
 @end direntry
 
 
