--- XaoS/src/ui/ui-drv/svga/ui_svga.c.orig	2004-11-24 08:11:48.000000000 +0100
+++ XaoS/src/ui/ui-drv/svga/ui_svga.c	2006-10-19 20:15:22.801637000 +0200
@@ -339,8 +339,6 @@
     }
 #endif
   vga_setmode (mode);
-  seteuid (getuid ());		/* Don't need supervisor rights anymore. */
-  setegid (getgid ());
   if (mode == G320x240x256 || mode == G320x400x256 || mode == G360x480x256)
     svga_driver.textheight = 8 * 4;
   else
@@ -458,29 +456,6 @@
   int i;
   int fd;
   struct stat chkbuf;
-  seteuid (euid);		/* We need supervisor rights to open mouse. */
-  setegid (egid);
-  if ((fd = open ("/dev/console", O_RDONLY)) < 0)
-    {
-      seteuid (getuid ());	/* Don't need supervisor rights anymore. */
-      setegid (getgid ());
-      return (0);
-    }
-  close (fd);
-  fstat (2, &chkbuf);
-  if (chkbuf.st_rdev >> 8 != 4 || (chkbuf.st_rdev & 0xff) >= 64)
-    {
-      seteuid (getuid ());	/* Don't need supervisor rights anymore. */
-      setegid (getgid ());
-      return (0);
-    }
-  if ((fd = open ("/dev/mem", O_RDWR)) < 0)
-    {
-      seteuid (getuid ());	/* Don't need supervisor rights anymore. */
-      setegid (getgid ());
-      return (0);
-    }
-  close (fd);
   if (!initialised)
     {
       vga_setmousesupport (1);
@@ -500,8 +475,6 @@
       svga_uninitialise ();
       return 0;
     }
-  seteuid (getuid ());		/* Don't need supervisor rights anymore. */
-  setegid (getgid ());
   return (1);
 }
 static void
